src/Core/src/Handlers/View/ViewHandler.cs

```diff
@@ -13,8 +13,16 @@
 
 namespace Microsoft.Maui.Handlers
 {
+	/// <summary>
+	/// Base class for handlers that manage views which implement <see cref="IView"/>.
+	/// </summary>
+	/// <remarks>Handlers map virtual views (.NET MAUI layer) to controls on each platform (iOS, Android, Windows, macOS, etc.), which are known as platform views.
+	/// Handlers are also responsible for instantiating the underlying platform view, and mapping the cross-platform control API to the platform view API. </remarks>
 	public abstract partial class ViewHandler : ElementHandler, IViewHandler
 	{
+		/// <summary>
+		/// A dictionary that maps the virtual view properties to their platform view counterparts.
+		/// </summary>
```

Comment:
I feel we can add a little more explanation for this, especially since it's one of the key concepts of the architecture, but no inspiration currently.

Reply:
I second this idea. Not very familiar with the property mapper so we might need someone more knowledgeable to help us out 😅